gradleops/AndroidSeed/tealeaf/build.gradle

```diff
@@ -0,0 +1,79 @@
+//<!--START_PLUGINS_PATCH-->
+//<!--END_PLUGINS_PATCH-->
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 27
+    buildToolsVersion '27.0.3'
+
+    defaultConfig {
+        minSdkVersion 19
+        targetSdkVersion 27
+        multiDexEnabled true
+        ndk {
+            moduleName "libpng"
+        }
+        versionCode 1
+        versionName "1.0"
+        lintOptions {
+            disable 'LongLogTag'
+            abortOnError false
+        }
+        
+    //<!--START_MANIFEST_PLACEHOLDERS-->
+    //<!--END_MANIFEST_PLACEHOLDERS-->
+        
+    }
+
+   useLibrary 'org.apache.http.legacy'
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+        }
+    }
+    sourceSets {
+        main {
+            jni.srcDirs = ['src/main/jni']
+            jniLibs.srcDirs = ['src/main/jni/lib', "src/main/libs", 'src/main/jni/libs']
+        }
+    }
+
+    externalNativeBuild {
+        ndkBuild {
+            path 'src/main/jni/Application.mk'
+        }
+    }
+    productFlavors {
+    }
+
+    allprojects {
+        repositories {
+            jcenter()
+            maven { url 'https://maven.google.com' }
+        }
+    }
+}
+
+dependencies {
+    implementation 'com.google.code.gson:gson:2.2.1'
+    implementation 'com.android.support:appcompat-v7:27.1.1'
```

Comment:
so app-compat-v7 is already included in app/build.gradle.
Should it be present in both ?
Trying to understand

Reply:
Yes

Reply:
got it, so best to retain it.